arcgis-ios-sdk-samples/Layers/RGB renderer/RGBRendererViewController.swift

```diff
@@ -42,7 +42,7 @@ class RGBRendererViewController: UIViewController, RGBRendererSettingsVCDelegate
     
     // MARK: - RGBRendererSettingsVCDelegate
     
-    func rgbRendererSettingsVC(_ rgbRendererSettingsVC: RGBRendererSettingsVC, didSelectStretchParameters parameters: AGSStretchParameters) {
+    func rgbRendererSettingsViewController(_ rgbRendererSettingsVC: RGBRendererSettingsViewController, didSelectStretchParameters parameters: AGSStretchParameters) {
```

Comment:
The name of the first parameter has changed in the protocol, so that should be reflected here.